build.sbt

```diff
@@ -4,7 +4,7 @@ pgpSecretRing := file(s"$gpgFolder/secring.gpg")
 
 lazy val commonDependencies: Seq[ModuleID] = Seq(
   %%("cats-core"),
-  %%("freestyle-async"),
+  %%("freestyle-async", "0.2.0"),
```

Comment:
Yeah, this is why I'm using 0.2.0 here. The existing macro is not compatible with scalameta, we can upgrade the freestyle version once we fix #8.